sessionCheckGlobal.js

```diff
@@ -65,6 +65,9 @@
      * the sessionCheckFrame code.
      */
     var idTokenRequest = function(config) {
+        if (!config.iframe) {
+            throw new Error("This session check instance has been destroyed")
```

Comment:
Instead of throwing an error, we should probably just use console.warn. I can see cases where people have event handlers still attempting to call triggerSessionCheck that they may not have removed, and it's possible errors thrown from here will break their code.

Reply:
I changed it to `console.warn` and added a `return`. This way it no longer throws an explicit error from having already destroyed the instance, but also doesn't continue execution down to the `.contentWindow.location.replace()` where it would also throw an exception due to the missing iframe.

Reply:
Great, thanks!